app/code/Magento/Checkout/Test/Mftf/Section/CheckoutCartProductSection.xml

```diff
@@ -48,6 +48,7 @@
         <element name="checkoutCartProductPrice" type="text" selector="//td[@class='col price']//span[@class='price']"/>
         <element name="checkoutCartSubtotal" type="text" selector="//td[@class='col subtotal']//span[@class='price']"/>
         <element name="emptyCart" selector=".cart-empty" type="text"/>
+        <element name="emptyCartButton" selector="#empty_cart_button" type="button"/>
         <!-- Required attention section -->
```

Comment:
add another element to support new actiongroup mentioned below (might need to change selector depending on where text is in modal)

`<element name="modalMessage" selector=".modal-popup.confirm._show .modal-content" type="text" timeout="30"/>`

Reply:
Added new elements for modal